VideoApi/VideoApi/appsettings.json

```diff
@@ -42,5 +42,5 @@
     "StorageEndpoint": "https://cvpstorageendpoint.net",
     "ManagedIdentityClientId": ""
   },
-  "UseStub": "false"
+  "UseStub": "true"
```

Comment:
revert? if not set in config when deployed it will use this setting from appsettings.json

Reply:
done